API.md

```diff
@@ -297,29 +297,16 @@ Renter
 
 Queries:
 
-* /renter/download
 * /renter/downloadqueue
-* /renter/files
-* /renter/file/delete
-* /renter/file/rename
-* /renter/file/share/save
-* /renter/file/share/load
-* /renter/upload
-
-#### /renter/download
-
-Function: Starts a file download.
-
-Parameters:
-```
-nickname    string
-destination string
-```
-`nickname` is the nickname of the file that has been uploaded to the network.
-
-`destination` is the path that the file will be downloaded to.
-
-Response: standard
+* /renter/files/delete
+* /renter/files/download
+* /renter/files/list
+* /renter/files/load
+* /renter/files/loadascii
+* /renter/files/rename
+* /renter/files/share
+* /renter/files/shareascii
+* /renter/files/upload
```

Comment:
it doesn't make sense to group actions on all files (`list`) with actions on single files.